main.py

```diff
@@ -1,17 +1,34 @@
 import argparse
+
+from colorama import init, Fore, Style
+
 from sdarot import SdarotPy
 from searchSerie import search
+from utils import center
+from configuration import Configuration
+
+init(autoreset=True)
 
 MAX_EP_NUM = 50
 
 if __name__ == "__main__":
+    print("\n\n")
+    print(Fore.GREEN + Style.BRIGHT + center(",d88~~\       888                             d8        888~-_           "))
+    print(Fore.GREEN + Style.BRIGHT + center("8888     e88~\888   /~~~8e  888-~\  e88~-_  _d88__      888   \  Y88b  / "))
+    print(Fore.GREEN + Style.BRIGHT + center("`Y88b   d888  888       88b 888    d888   i  888   ____ 888    |  Y888/  "))
+    print(Fore.GREEN + Style.BRIGHT + center(" `Y88b, 8888  888  e88~-888 888    8888   |  888        888   /    Y8/   "))
+    print(Fore.GREEN + Style.BRIGHT + center("   8888 Y888  888 C888  888 888    Y888   '  888        888_-~      Y    "))
+    print(Fore.GREEN + Style.BRIGHT + center("\__88P'  \"88_/888  \"88_-888 888     \"88_-~   \"88_/      888        /     "))
+    print(Fore.GREEN + Style.BRIGHT + center("                                                                 _/      "))
```

Comment:
one last thing for non unix users, to print this properly on cmd 
```
    print(Fore.GREEN + Style.BRIGHT + center(",d88~~\       888                             d8        888~-_           "), end='')
    print(Fore.GREEN + Style.BRIGHT + center("8888     e88~\888   /~~~8e  888-~\  e88~-_  _d88__      888   \  Y88b  / "), end='')
    print(Fore.GREEN + Style.BRIGHT + center("`Y88b   d888  888       88b 888    d888   i  888   ____ 888    |  Y888/  "), end='')
    print(Fore.GREEN + Style.BRIGHT + center(" `Y88b, 8888  888  e88~-888 888    8888   |  888        888   /    Y8/   "), end='')
    print(Fore.GREEN + Style.BRIGHT + center("   8888 Y888  888 C888  888 888    Y888   '  888        888_-~      Y    "), end='')
    print(Fore.GREEN + Style.BRIGHT + center("\__88P'  \"88_/888  \"88_-888 888     \"88_-~   \"88_/      888        /     "), end='')
    print(Fore.GREEN + Style.BRIGHT + center("                                                                 _/      "), end='')
```

Reply:
are you using windows ?
can you check to omit the center and see if it resolve the issue for you ? it might be that it works only on mac / linux
i'll add a fix for it, please check it out as i do not run on windows

Reply:
its solves it but its not in the center, adding end='' solves it (without removing center)